src/layouts/Footer.jsx

```diff
@@ -1,7 +1,15 @@
 import React from 'react';
```

Comment:
Importing React here is not required so you can remove it.

Reply:
Linter requires it
error  'React' must be in scope when using JSX

Reply:
Evgenii is correct. Although I was able to bypass this by turning the rule off in eslint.json. Please advise if correct 
![Screenshot 2024-11-21 at 2 22 44 PM](https://github.com/user-attachments/assets/9eeb44f5-6746-4507-bb03-553f2fa50bf2)